src/System.Data.SqlClient/src/System/Data/SqlClient/LocalDBAPI.Common.cs

```diff
@@ -102,7 +102,7 @@ private static SqlException CreateLocalDBException(string errorMessage, string i
             if (sniError != 0)
             {
                 string sniErrorMessage = SQL.GetSNIErrorMessage(sniError);
-                errorMessage = string.Format((IFormatProvider)null, "{0} (error: {1} - {2})",
+                errorMessage = string.Format("{0} (error: {1} - {2})",
```

Comment:
Use string interpolation?

Reply:
We could in this case, and it should result in exactly the same codegen.  I've just shied away from doing that in general.